Code/Source/RobotImporter/URDF/URDFPrefabMaker.cpp

```diff
@@ -146,4 +151,28 @@ namespace ROS2
     {
         return m_prefabPath;
     }
+
+    void URDFPrefabMaker::MoveEntityToDefaultSpawnPoint(const AZ::EntityId& rootEntityId)
+    {
+        auto spawner = ROS2::SpawnerInterface::Get();
+
+        if (spawner == nullptr)
+        {
+            AZ_TracePrintf("URDF Importer", "Spawner not found - creating entity in (0,0,0)") return;
+        }
+
+        auto entity_ = AzToolsFramework::GetEntityById(rootEntityId);
+        auto* transformInterface_ = entity_->FindComponent<AzToolsFramework::Components::TransformComponent>();
+
+        if (transformInterface_ == nullptr)
+        {
+            AZ_TracePrintf("URDF Importer", "TransformComponent not found in created entity") return;
+        }
+
+        AZ::Vector3 pos = spawner->GetDefaultSpawnPoint();
+
+        auto transform = AZ::Transform(pos, AZ::Quaternion(0.0f, 0.0f, 0.0f, 1.0f), 1.0f);
```

Comment:
I am wondering whether default spawn point should actually be a default spawn pose.. Since it is important with robots which cannot turn in place. 
Could you propose a solution?